tensorflow_lite_support/python/test/task/vision/image_classifier_test.py

```diff
@@ -128,6 +164,129 @@ def test_classify_model(self, model_file_type, max_results,
+    image_result_dict = json.loads(json_format.MessageToJson(image_result))
+
+    # Expected results.
+    expected_categories = [
+      {'index': 934, 'score': 0.8815076351165771, 'class_name': "cheeseburger"},
+      {'index': 925, 'score': 0.019456762820482254, 'class_name': "guacamole"},
+      {'index': 932, 'score': 0.012489477172493935, 'class_name': "bagel"}
+    ]
+
+    # Builds test data.
+    expected_result_dict = self.build_test_data(expected_categories)
+
+    # Comparing results (classification w/ bounding box).
+    self.assertDeepAlmostEqual(image_result_dict, expected_result_dict, places=5)
+
+  def test_score_threshold_option(self):
+    # Creates classifier.
+    model_file = _ExternalFile(file_name=self.model_path)
+
+    classifier = self.create_classifier_from_options(
+      model_file, score_threshold=0.5)
+
+    # Loads image.
+    image = tensor_image.TensorImage.from_file(self.test_image_path)
+
+    # Classifies the input.
+    image_result = classifier.classify(image, bounding_box=None)
+    image_result_dict = json.loads(json_format.MessageToJson(image_result))
+
+    # Expected results.
+    expected_categories = [
+      {'index': 934, 'score': 0.7399742007255554, 'class_name': "cheeseburger"}
+    ]
+
+    # Builds test data.
+    expected_result_dict = self.build_test_data(expected_categories)
+
+    # Comparing results.
+    self.assertDeepAlmostEqual(image_result_dict, expected_result_dict, places=5)
+
+  def test_allowlist_option(self):
+    # Creates classifier.
+    model_file = _ExternalFile(file_name=self.model_path)
+
+    classifier = self.create_classifier_from_options(
+      model_file, class_name_allowlist=['cheeseburger', 'guacamole'])
+
+    # Loads image.
+    image = tensor_image.TensorImage.from_file(self.test_image_path)
+
+    # Classifies the input.
+    image_result = classifier.classify(image, bounding_box=None)
+    image_result_dict = json.loads(json_format.MessageToJson(image_result))
+
+    # Expected results.
+    expected_categories = [
+      {'index': 934, 'score': 0.7399742007255554, 'class_name': "cheeseburger"},
+      {'index': 925, 'score': 0.026928534731268883, 'class_name': "guacamole"}
+    ]
+
+    # Builds test data.
+    expected_result_dict = self.build_test_data(expected_categories)
+
+    # Comparing results.
+    self.assertDeepAlmostEqual(image_result_dict, expected_result_dict, places=5)
+
+  def test_denylist_option(self):
+    # Creates classifier.
+    model_file = _ExternalFile(file_name=self.model_path)
+
+    classifier = self.create_classifier_from_options(
+      model_file, score_threshold=0.01, class_name_denylist=['cheeseburger'])
+
+    # Loads image
+    image = tensor_image.TensorImage.from_file(self.test_image_path)
+
+    # Classifies the input.
+    image_result = classifier.classify(image, bounding_box=None)
+    image_result_dict = json.loads(json_format.MessageToJson(image_result))
+
+    # Expected results.
+    expected_categories = [
```

Comment:
Same as above. as you're testing the deny, you should assert if none of the class names in image_result belongs to the class_name_denylist, rather than comparing it with a predefined result.